pkg/controller/endpointslice/reconciler_test.go

```diff
@@ -177,6 +217,19 @@ func TestReconcile1Pod(t *testing.T) {
 				t.Errorf("Expected EndpointSlice to have label set with %s value, got %s", testCase.service.Name, slice.Labels[discovery.LabelServiceName])
 			}
 
+			// slice inherit parent service labels but reserved ones
```

Comment:
This comment is a bit confusing? Maybe `Slices should inherit labels from parent Service unless they are reserved`?

Reply:
sure